pages/Game.js

```diff
@@ -1,18 +1,103 @@
 import React from 'react';
-import Grid from './Grid';
+import Grid from '../src/game/Grid';
+import RemainingList from '../src/game/RemainingList';
+import ActionText from '../src/game/ActionText';
+import Link from 'next/link';
+import Button from '../src/ui/Button';
+import Container from '../src/ui/Container';
+import config from '../config/config.dist';
+import PropTypes from 'prop-types';
+import fetch from 'isomorphic-unfetch';
+import io from 'socket.io-client';
+import styled from 'react-emotion';
+
+const BoardContainer = styled('div')`
+    height: 500px;
+`;
 
 class Game extends React.Component {
+    state = {
+        game: {},
+        loaded: false,
+    };
+
+    static propTypes = {
+        game: PropTypes.object.isRequired,
+        loaded: PropTypes.bool,
+    };
+
+    static async getInitialProps() {
+        let url = `http://${config.apiUrl}/479`;
```

Comment:
The url could be a constant defined at the root of the file (to avoid useless computation).